docs/content/latest/explore/linear-scalability/scaling-transactions.md

```diff
@@ -80,44 +99,61 @@ The workload application prints some statistics while running, an example is sho
 
-![Read and write IOPS with 3 nodes](/images/ce/linear-scalability-3-nodes.png)
+![Read and write IOPS with 3 nodes](/images/ce/transactions_observe.png)
 
 ## 4. Add node and observe linear scaling
 
-Add a node to the universe.
+Add a node to the universe with the same flags.
 
 ```sh
-$ ./bin/yb-ctl --num_shards_per_tserver 4 add_node
+$ ./bin/yugabyted start \
+                  --base_dir=/tmp/ybd4 \
+                  --listen=127.0.0.4 \
+                  --join=127.0.0.1 \
+                  --master_flags "ysql_num_shards_per_tserver=4" \
+                  --tserver_flags "ysql_num_shards_per_tserver=4,follower_unavailable_considered_failed_sec=30"
 ```
 
 Now you should have 4 nodes. Refresh the <a href='http://127.0.0.1:7000/tablet-servers' target="_blank">tablet-servers</a> page to see the statistics update. Shortly, you should see the new node performing a comparable number of reads and writes as the other nodes. The 36 tablets will now get distributed evenly across all the 4 nodes, leading to each node having 9 tablets.
 
 The YugabyteDB universe automatically lets the client know to use the newly added node for serving queries. This scaling out of client queries is completely transparent to the application logic, allowing the application to scale linearly for both reads and writes.
 
-![Read and write IOPS with 4 nodes - Rebalancing in progress](/images/ce/linear-scalability-4-nodes.png)
+![Read and write IOPS with 4 nodes - Rebalancing in progress](/images/ce/transactions_newnode_adding_observe.png)
 
-![Read and write IOPS with 4 nodes](/images/ce/linear-scalability-4-nodes-balanced.png)
+![Read and write IOPS with 4 nodes](/images/ce/transactions_newnode_added_observe.png)
 
 ## 5. Remove node and observe linear scale in
 
 Remove the recently added node from the universe.
 
 ```sh
-$ ./bin/yb-ctl remove_node 4
+$ ./bin/yugabyted stop \
+                  --base_dir=/tmp/ybd4
 ```
 
-Refresh the <a href='http://127.0.0.1:7000/tablet-servers' target="_blank">tablet-servers</a> page to see the stats update. The `Time since heartbeat` value for that node will keep increasing. Once that number reaches 60s (i.e. 1 minute), YugabyteDB will change the status of that node from ALIVE to DEAD. Note that at this time the universe is running in an under-replicated state for some subset of tablets.
+Refresh the <a href='http://127.0.0.1:7000/tablet-servers' target="_blank">tablet-servers</a> page to see the stats update. The `Time since heartbeat` value for that node will keep increasing. Once that number reaches 60s (i.e. 1 minute), YugabyteDB will change the status of that node from ALIVE to DEAD. We can now see the load (tablets) and IOPS getting moved off the removed node and redistributed amongst the other nodes.
```

Comment:
```suggestion
Refresh the <a href='http://127.0.0.1:7000/tablet-servers' target="_blank">tablet-servers</a> page to see the stats update. The `Time since heartbeat` value for that node will keep increasing. Once that number reaches 60s (i.e. 1 minute), YugabyteDB will change the status of that node from ALIVE to DEAD. Observe the load (tablets) and IOPS getting moved off the removed node and redistributed amongst the other nodes.
```